docs/kubernetes.md

```diff
@@ -32,6 +32,34 @@ In the image above, you can see the following parts:
 
 All VMs are in the same private VNET and are fully accessible to each other.
 
+
+## Optional: Enable network policy enforcement using Calico

```

Comment:
I agree that it's not optimal that you are not aware of the parameters at creation time of a cluster. I guess that's a general problem for all possible parameters. I added a note and cross reference to the calico section. Let me know, if you think that's appropriate